tests/director/tests/formation_api_test.go

```diff
@@ -88,8 +90,6 @@ func TestListFormations(t *testing.T) {
 	listFormationsReq := fixtures.FixListFormationsRequestWithPageSize(first)
 	saveExample(t, listFormationsReq.Query(), "query formations")
 	formationPage1 := fixtures.ListFormations(t, ctx, certSecuredGraphQLClient, listFormationsReq, expectedFormations)
-	require.NotNil(t, formationPage1)
```

Comment:
This check is unnecessary because if you look at the `fixtures.ListFormations` function we have `require.NotEmpty(t, formationPage)` and `require.Equal(t, expectedCount, formationPage.TotalCount)` where not empty check validates not nil as well, from the go doc `NotEmpty asserts that the specified object is NOT empty.  I.e. not nil, "", false, 0 or ...`
So this will be handled in the function as well as the count and here I think we do not need this?